script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -241,12 +261,12 @@ I realize that I have, right now, with my own hands, brought down the curtain on
 [sha:848dfeae88ab7d3ae0ad489fc6fde5b194d77d97]{
 -- Page 372, Offset 28077.
 -- それは確かな事だと、俺も思います」
-I believe this is true."
+I don't doubt that she was telling the truth."
```

Comment:
>I have no doubt...

Is more normal to me, but I genuinely don't know

Reply:
Don't doubt more colloquial I feel

Reply:
I feel like Ciel is a bit more proper.

Reply:
Shiki's talking here though

Reply:
Hmm, actually next line starts with `I don't` as well, so changed anyway